src/vivarium_dcpn_hypertension_mgt/components/hypertension_medication.py

```diff
@@ -71,4 +71,48 @@ def probability(self, index: pd.Index):
             filtered_index = filtered_index.intersection(bp_index)
 
         p.loc[filtered_index] = super().probability(filtered_index)
-        return p
\ No newline at end of file
+        return p
+
+
+class CVDRiskAttribute:
+
+    @property
+    def name(self):
+        return 'cvd_risk_attribute'
+
+    def setup(self, builder):
+
+        self.bmi = builder.value.get_value('high_body_mass_index_in_adults.exposure')
+        self.fpg = builder.value.get_value('high_fasting_plasma_glucose_continuous.exposure')
+
+        self.population_view = builder.population.get_view(['ischemic_heart_disease_event_time'])
+
+        self.clock = builder.time.clock()
+
+        builder.value.register_value_producer('cvd_risk_category', source=self.get_cvd_risk_category)
+
+    def get_cvd_risk_category(self, index):
+        """CVD risk category is 0 if bmi and fpg are not within risk ranges and
+        there has not been an ihd event in the last year; 1 otherwise.
+
+        Risk Ranges:
+            bmi >= 28 km/m2
```

Comment:
think this is kg here :D

Reply:
lol yes. changed. thanks for catching that